### PR TITLE
Convert the Formula conformance suite tests to pytest.

### PR DESCRIPTION
**Pull request by stevenbronson-wk**
#### Steps to Test
CI

**review**:
@Arelle/arelle

### CODE REVIEW

File: tests/integration_tests/validation/formula/test_formula_conformance_suite.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+
+TEST_DATA = get_test_data(ARGS)
+
+
+@pytest.mark.parametrize("result", TEST_DATA)
+def test_formula(result, request):
+    bad_id = '30000 Assertions/31140-ConsistencyAssertion-StaticAnalysis-missingFormulae/V-01'
+    if request.node.callspec.id == bad_id:
+        pytest.xfail(f"missing formula test '{bad_id}' doesn't work yet")
```
Comment (derekgengenbacher-wf) [inline tests/integration_tests/validation/formula/test_formula_conformance_suite.py:25]: Nice

**Review by derekgengenbacher-wf — APPROVED** [review]

**Review by austinmatherne-wk — DISMISSED** [review]

**Review by sagesmith-wf — DISMISSED** [review]

**Review by hermfischer-wf — CHANGES_REQUESTED** [review]
The formula tests are in several parts, of which #1 is implemented:

# Formula main suite (entry index.xml in formula suite)
 *  may need `--plugin formulaXPathChecker --check-formula-restricted-XPath`

# Formula Registry suite (entry function-registry/registry-index.xml)
 * may need `--plugin formulaXPathChecker --check-formula-restricted-XPath`

# Specific subsuites under tests/60000 Extension (which may not be covered by top level index.xml
 * 60100 GenericMessages-Processing
 * 60110 AssertionSeverity-Processing
 * 60111 AssertionSeverity-2.0-Processing
 * 60200 CustomXPathFuctions-Processing
 * 61000 AspectCoverFilter-Processing
 * 61100 ConceptRelationsFilter-Processing
 * 70000 Generic Linkbase tests indexed by index-gnl.xml

**Comment by aviary2-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

**Review by austinmatherne-wk — APPROVED** [review]

**Review by sagesmith-wf — DISMISSED** [review]

**Review by sagesmith-wf — APPROVED** [review]

**Review by aaroncameron-wk — APPROVED** [review]

**Review by austinmatherne-wk — APPROVED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 CI succeeded.

**Pull request merged**